packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -10,6 +10,7 @@
 }
 
 .card__link--image {
+  contain: unset;
```

Comment:
I was about to talk to the browser compatibility of `unset`, but looks like IE 11 was really the only sticking point that's kept us from using it in the past. Nice!